openmls/src/error.rs

```diff
@@ -121,3 +112,29 @@ impl ErrorString {
         self.0.clone()
     }
 }
+
+/*
+
+Note to maintainers
+
+The thiserror crate does not add any documentation to enum variants and this needs to be done manually.
+The best way to do this if you don't want to duplicate the string manually is to use the following regex:
+
+Add comments for naked variants:
+
+ ([,|{])\n(\s+)#\[error\("([a-z0-9 .,-_'^:]+)"\)\]
```

Comment:
This is certainly useful for the future. But can you add some indication how you applied this? Bash script? Python? The regex is certainly useful, but it would be more useful if I know how to interpret the $1..3, etc. below.

Reply:
Also, the wiki would probably be a better place for this.

Reply:
Re usage: I guess that depends on your favourite editor. In VS Code you can just copy & paste this.

Re wiki: Not sure what will happen to the wiki going forward. After we clean it up, not much will be left there. I thought it was safer here.

Reply:
Re usage: Then it would be useful to add the information that this is VS Code specific.

Re wiki: Fair point. Leave it here for now, then.